rocAL/rocAL/source/master_graph.cpp

```diff
@@ -1511,3 +1511,53 @@ MasterGraph::copy_out_tensor_planar(void *out_ptr, RaliTensorFormat format, floa
     _convert_time.end();
     return Status::OK;
 }
+
+void *
+MasterGraph::copy_hip_out_tensor(RaliTensorFormat format, float multiplier0, float multiplier1,
```

Comment:
todo:: we need to have only one function which should work for both host and device